.travis.yml

```diff
@@ -1,82 +0,0 @@
-language: rust
-dist: bionic
-os: linux
-
-branches:
-  only:
-  - auto
-  - try
-  - master
-
-cache:
-  cargo: true
-  yarn: true
-
-before_cache: script/ci/prune-cache.sh
-
-env:
-  global:
-    - RUST_BACKTRACE=1
-    - JOBS=1 # See https://git.io/vdao3 for details.
-    - DATABASE_URL=postgres:///cargo_registry_test
-    - TEST_DATABASE_URL=postgres:///cargo_registry_test
-    - CARGO_INCREMENTAL=0
-    # Percy secrets are included here to enable Percy's GitHub integration
-    # on community-submitted PRs
-    - PERCY_TOKEN=0d8707a02b19aebbec79bb0bf302b8d2fa95edb33169cfe41b084289596670b1
-    - PERCY_PROJECT=crates-io/crates.io
-    - PATH=$HOME/.cargo/bin:$PATH
-    - RUSTFLAGS="-C debuginfo=0 -D warnings"
-    - MALLOC_CONF=background_thread:true,abort_conf:true,abort:true,junk:true
-
-install:
-  - script/ci/cargo-clean-on-new-rustc-version.sh
-  - which diesel || cargo install diesel_cli --vers `cat .diesel_version` --no-default-features --features postgres --debug
-
-before_script:
-  - diesel database setup --locked-schema
-
-addons:
-  postgresql: "11"
-  apt:
-    packages:
-      - postgresql-11
-      - postgresql-client-11
-
-jobs:
-  fast_finish: true
-  allow_failures:
-    - rust: nightly
```

Comment:
As far as I'm aware, this is the only gap between our Travis and GH Actions configurations. On Actions we don't currently test on nightly because there wasn't an easy way to replicate the `allow_failures` functionality. I don't think this is a big deal, but if there's an easy way to do this in actions now that would be nice.

Reply:
There is no direct equivalent, but if we don't mind the red "x" we can just put the nighly build in a job not needed by `bors build finished`.

Reply:
> if we don't mind the red "x"

That would look quite badly all over the GitHub UI. We could use `cargo test || true`, but then we would never see it if nightly actually starts failing.

I'm wondering if `cargo test || echo "::warning ::nightly build failed"` would work and show up in the UI 🤔